Makefile

```diff
@@ -106,7 +106,7 @@ uninstall: manifests
 
 deploy: ## Deploy the operator to a Kubernetes cluster. This assumes a kubeconfig in ~/.kube/config
 deploy: manifests
-	cd config/manager && kustomize edit set image contour-operator=${IMAGE}:${VERSION}
+	cd config/manager
 	kustomize build config/default | kubectl apply -f -
```

Comment:
@stevesloka could be removed with some additional work. Kustomize is still used to build the example operator manifest. I'd prefer to use Kustomize for this instead of a script. I think Kustomize may also be useful to support customizing the operator manifest for future requirements.